tests/pyright/test_fields.py

```diff
@@ -121,28 +97,16 @@ class Foo(models.Model):
     assert results == [
         Result(
             type="info",
-            message="Type of \"f.char_with_choices\" is \"Literal['a', 'b']\"",
+            message='Type of "f.char_with_choices" is "str"',
             line=12,
             column=13,
         ),
-        Result(
-            type="error",
-            message="Condition will always evaluate to False since the types \"Literal['a', 'b']\" and \"Literal['c']\" have no overlap (reportUnnecessaryComparison)",
-            line=13,
-            column=1,
-        ),
         Result(
             type="error",
             message='Cannot assign member "char_with_choices" for type "Foo" (reportGeneralTypeIssues)',
             line=15,
             column=3,
         ),
-        Result(
-            type="error",
-            message='Cannot assign member "char_with_choices" for type "Foo" (reportGeneralTypeIssues)',
```

Comment:
curious if these changes are expected, seems were losing some strictness with the changes

Reply:
We are actually just loosing the strictness that I previously added for the "Literal" capturing of the `choices` attributes. Other than that, the change should not have removed any strictness